### DISCUSSION TITLE
API_GITHUB_KEY = "client secret" key or "private key"?

### DISCUSSION THREAD

**Discussion by spet0683** [OpenBB-finance/OpenBB#6310] [Q&A]
I'm trying to set an API key for Github within OpenBB.  I've tried entering a "client secret" key..."defined, not tested".  I've tried entering the "private key"...also "defined, not tested".  From what I can gather I'm inputting the wrong thing or inputting the correct thing with the incorrect syntax. How does one do this successfully?

**Comment by markoblogo** [comment]
You don’t need to paste any GitHub “client secret” or “private key” into API_GITHUB_KEY for OpenBB.
The GitHub integration just relies on your local git setup: as long as you can run git clone https://github.com/OpenBB-finance/OpenBB.git (or git pull in that repo) without errors, the update command will work and no extra GitHub API key is required.
If you previously added something under the GitHub key in the OpenBB settings, you can safely clear it and just make sure your global Git credentials (or SSH key) are configured correctly in Git instead.
